File: python-sdk/examples/trending-news-source-impact.py
'''

	Endpoint: public/trending-news-source-impact
	Name: Trending News Source Impact

'''
import sys
import json

sys.path.insert(1, '../src')

# Include config and class
from config import key, secret
from Class import ApiAccess

# Declare your key and secret
obj = ApiAccess(key, secret)

# Declare endpoint
endpoint = 'public/trending-news-source-impact'

# Only one parameter
payload = {'source_contains':''}

result = obj.send(endpoint, payload, dict())

print(json.dumps(result['returned'], indent=4))

File: php-sdk/examples/trending-news-source-impact.php
<?php

/*

	File: trending-news-source-impact.php
	Endpoint: public/trending-news-source-impact
	Repo: https://github.com/executium/trending-historical-cryptocurrency-news
	License: https://github.com/executium/trending-historical-cryptocurrency-news/LICENSE

*/

# Include the executium class
include '../src/Executium.trendingnews.Class.php';

# Declare your key and secret
$obj = new ExecutiumTrendingNews();

# Parameters required
$payload=array();
$payload['source_contains']='';

# Send Request
$result = $obj->trending_news_source_impact($payload);

# Result
echo '<pre>';
	echo "\n<strong>Returned</strong>\n\n";
	print_r($result);
echo '</pre>';

?>
